src/components/molecules/chat-tool/chat-room-list/ChatRoomList.tsx

```diff
@@ -1,26 +1,48 @@
 import React, { useState, useEffect } from 'react';
 import * as S from './styled';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { chatRoomAtom } from '@/recoil/atoms/ChatRoomAtom';
+import { useRouter } from 'next/router';
+import useNavigator from '@/hooks/useNavigator';
+import { MyProfileAtom } from '@/recoil/atoms/MyProfileAtom';
 
 const ChatRoomList = () => {
-  const [listSelected, setListSelected] = useState(false);
-  const handleListSelected = () => {
-    setListSelected(true);
+  const navigateList = useNavigator();
+  const [chatRooms, setChatRooms] = useRecoilState(chatRoomAtom);
+  const [selectedRoomId, setSelectedRoomId] = useState<string | null>(null);
```

Comment:
```suggestion
  const [selectedRoomId, setSelectedRoomId] = useState(null);
```
초기값 할당인데, 타입 명시 안해주면 오류 나나요?

Reply:
옙 오류납니다잉 selectedRoomId는 chatRoomList들 중 각각의 chatRoom을 선택(클릭)했는지에 대해서만 관리하는 상태이기 때문에, boolean 값으로도 할 수 있다고 판단했으나 실질적으로 chatsRooms의 변동으로 상태가 변한다고 판단해서 string값과 아예 값이 없는 null값으로 타입을 명시해주었습니다.